homeassistant/components/linear_garage_door/config_flow.py

```diff
@@ -80,6 +72,9 @@ async def async_step_user(
         if user_input is None:
             return self.async_show_form(step_id="user", data_schema=data_schema)
 
+        await self.async_set_unique_id(user_input[CONF_EMAIL])
```

Comment:
Is the email unique?

Reply:
I believe so yeah, I could call `.lower()` or `.upper()` on it if you'd like